package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupyterlab/github",
-  "version": "3.0.1",
+  "version": "4.0.0",
```

Comment:
Should we revert the version back to `3.0.1` here for now?

When we will be making the release, we can specify `4.0.0` to the releaser. Otherwise if the version is already `4.0.0`, the releaser will not allow setting the same version (`4.0.0`).

Reply:
True that!